docs/source/release_notes.rst

```diff
@@ -12,6 +12,7 @@ Release Notes
         * Replaced `allowed_pipelines` with `allowed_component_graphs` :pr:`2364`
         * Removed private method ``_compute_features_during_fit`` from ``PipelineBase`` :pr:`2359`
         * Unpinned PyZMQ version in requirements.txt :pr:`2389` 
+        * Uncapping LightGBM version in requirements.txt :pr:`2405` 
```

Comment:
This is a mega nit-pick which you can ignore for this PR: we have tried to phrase our release notes entries in past-tense. "Uncapped LightGBM version"